cmd/cdi-importer/importer.go

```diff
@@ -103,3 +113,19 @@ func main() {
 	}
 	glog.V(1).Infoln("import complete")
 }
+
+func getAvailableSpace(volumeMode string) int64 {
+	if volumeMode == "Block" {
```

Comment:
So looks like fstat (used in util.GetAvailableSpace) takes the path to a block device and returns information. So if I understand the fields right from [here](https://golang.org/pkg/syscall/#Stat_t) BlkSize * Blocks should give you the available space, so you should be able to make a util.GetAvailableSpaceBlock space that looks like this:
```go
// GetAvailableSpaceBlock gets the amount of available space at the path specified.
func GetAvailableSpaceBlock(path string) int64 {
	var stat syscall.Statfs_t
	err := syscall.Statfs(path, &stat)
	if err != nil {
		return int64(-1)
	}
	return int64(stat.BlkSize) * int64(stat.Blocks)
}
```
Note all of this is untested and could not be right, but looking at the documentation this appears it should give us what we need.

Reply:
done